gulp/aliases.js

```diff
@@ -22,7 +22,8 @@ module.exports = (gulp) => {
     gulp.task("build", (done) => rs("clean", "compile", "webpack-compile-docs", done));
 
     // build code, run unit tests, terminate
-    gulp.task("test", ["test-dist", "karma", "isotest"]);
+    // NOTE: circle.yml runs each of these separately for better error reporting; be sure to sync
```

Comment:
means keep the two uses in sync. make sure they run the same tasks.

https://circleci.com/gh/palantir/blueprint/1503 shows two of the three failing.

i have an idea how to post comments on build failure, and we could post a different comment based on which ones fail.